gnovm/pkg/gnomod/utils.go

```diff
@@ -34,3 +37,16 @@ func FindRootDir(absPath string) (string, error) {
 
 	return "", ErrGnoModNotFound
 }
+
+func createGnoModPkg(t *testing.T, dirPath, pkgName, modData string) {
+	t.Helper()
+
+	// Create package dir
+	pkgDirPath := filepath.Join(dirPath, pkgName)
+	err := os.MkdirAll(pkgDirPath, 0o755)
+	require.NoError(t, err)
+
+	// Create gno.mod
+	err = os.WriteFile(filepath.Join(pkgDirPath, "gno.mod"), []byte(modData), 0o644)
+	require.NoError(t, err)
+}
```

Comment:
```suggestion
```

Already defined in pkg_test.go, and not (yet) used by an external package.

Reply:
separate PR please